tests/meshcop/commissioner.cpp

```diff
@@ -364,91 +336,85 @@ static void HandleCommissionerPetition(const Coap::Message &aMessage, void *aCon
         tlv = tlv->GetNext();
     }
 
+    gettimeofday(&commissioner->mLastKeepAliveTime, NULL);
     otbrLog(OTBR_LOG_INFO, "COMM_PET.rsp: complete");
 }
 
-/** Send the c/cp request to the border router agent */
-static int CommissionerPetition(Context &aContext)
+int Commissioner::CommissionerSet()
```

Comment:
style: "All void functions or methods shall explicitly declare and specify the void type keyword."